service/src/main/java/bio/terra/pipelines/dependencies/gcs/GcsClient.java

```diff
@@ -0,0 +1,16 @@
+package bio.terra.pipelines.dependencies.gcs;
+
+import com.google.cloud.storage.Storage;
+import com.google.cloud.storage.StorageOptions;
+import org.springframework.stereotype.Component;
+
+@Component
+public class GcsClient {
+
+  public GcsClient() {}
+
+  public Storage getStorageService(String projectId) {
+    // this will use application default credentials
```

Comment:
we should probably explicitly use the SA creds, right?

Reply:
application default credentials are set as the SA creds already, we do that [here](https://github.com/DataBiosphere/terra-scientific-pipelines-service/blob/main/service/build.gradle#L118)